modules/tinymce/src/themes/silver/main/ts/ui/dialog/ColorInput.ts

```diff
@@ -10,8 +10,8 @@ import { UiFactoryBackstageShared } from '../../backstage/Backstage';
 import { UiFactoryBackstageForColorInput } from '../../backstage/ColorInputBackstage';
 import * as ReadOnly from '../../ReadOnly';
 import { renderLabel } from '../alien/FieldLabeller';
+import { addColor } from '../core/color/ColorCache';
```

Comment:
Likewise here, as we always prefer using star imports generally to make it more readable.